Applications/Calculator/Calculator.cpp

```diff
@@ -28,6 +28,9 @@
 #include <AK/Assertions.h>
 #include <math.h>
 
+const double MAX = 2147483647;
+const double MIN = -2147483647;
```

Comment:
What are those numbers? Where do they come from? If it's something like `INT_MAX`, why not just use that?

And please use `constexpr` or `#define` or `enum` for constant primitives — that way, they'll be inlined instead of generating global variables.

Reply:
INT_MAX was what I was looking for. Fixed.